code-lens-provider-plugin/src/code-lens-provider-plugin-frontend-plugin.ts

```diff
@@ -0,0 +1,66 @@
+/**
```

Comment:
it seems it is missing copyright headers

Reply:
added. Thanks

Reply:
Should we have copyright headers for all our samples?
I forgot to add it in `tree-view-sample-plugin`

Reply:
I would say yes (we should have a tslint rule for that)

Reply:
It looks we haven't. We cannot build our samples from the root.

Reply:
ok maybe let's comment into a new issue

if you want to use lerna, etc to have isolated and also global commands